companies/onlinetvrecorder.json

```
@@ -0,0 +1,26 @@
+{
+    "slug": "onlinetvrecorder",
+    "relevant-countries": [
+        "de"
+    ],
+    "categories": [
+        "entertainment"
+    ],
+    "name": "Internet TV AG",
+    "runs": [
+        "onlinetvrecorder.com"
+    ],
+    "address": "306 Victoria House\n361 Victoria\nSeychelles\n\n",
```

```suggestion
    "address": "306 Victoria House\nVictoria Mahe\nSeychelles",
```
fixed formatting; see [https://www.onlinetvrecorder.com/v2/contact/gdpr](https://www.onlinetvrecorder.com/v2/contact/gdpr)